CriticalMass/Generated/Images.swift

```diff
@@ -2,11 +2,11 @@
 // Generated using SwiftGen — https://github.com/SwiftGen/SwiftGen
 
 #if os(macOS)
-    import AppKit
```

Comment:
What happened to the formatting in the generated files? Should they use 4 white spaces or 2 as indentation?

Reply:
The `Generated` folder was not excluded from `SwiftFormat` which runs in every build...
I'll fix it

Reply:
But in general I think that those are files we can ignore when it comes to coding conventions since they are autoGenerated

Reply:
the checked in file on `master` now is formatted and this one is not and will also add an `exclude` flag to the swiftformat yaml. Another option is to autogenerate the files and then execute swiftFormat after that. From a script or something like https://github.com/yonaskolb/Beak

Reply:
Ah, yeah... I don't think that the generated files should play any role in code conventions, but it's a good idea to exclude it for swiftformat so it's not "reformatted" every time. Was just curious about the changes here..